cerberus-web/src/main/java/com/nike/cerberus/jwt/CerberusJwtClaims.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2021 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.jwt;
+
+import com.nike.cerberus.domain.AuthTokenInfo;
+import java.time.OffsetDateTime;
+
+public class CerberusJwtClaims implements AuthTokenInfo {
```

Comment:
Could definitely use the Lombak getters and setters in this class

Reply:
I'll add Getters